### ISSUE TITLE
AD:namu.wiki

### ISSUE THREAD

**Issue by judah69** [List-KR/NamuLink#30]
![IMG_8395](https://github.com/List-KR/NamuLink/assets/158001645/9051da25-ced0-43d8-8240-f223956dce4d

### URL

https://namu.wiki/w/백숙

### FILTERS

adguard base)

adguard mobile

youslist

list kr

adguard dns

### Browser

Apple Safari

### Operating System

Apple iOS

### Ad-Blocker

AdGuard for iOS/iOS Pro

### Userscript Manager

quoid/userscripts

### USERSCRIPT

namurink

### Comment

As shown in the picture, a Namu Wiki power link will appear at the bottom.

**Comment by piquark6046** [comment]
Please send a screenshot.

**Comment by judah69** [comment]
![IMG_8395](https://github.com/List-KR/NamuLink/assets/158001645/1f07ec7c-45ff-4193-9e94-e0d0604937e3)

**Comment by piquark6046** [comment]
Do you have a macOS device?

**Comment by judah69** [comment]
no only use iphone

**Comment by piquark6046** [comment]
When does the advert appear?
Navigate an article?

**Comment by judah69** [comment]
When search on Namu Wiki and when search on Google, both ads appear

**Comment by piquark6046** [comment]
Please check that NamuLink is up-to-date.

**Comment by judah69** [comment]
I reinstalled it, but ads still appear.

**Comment by piquark6046** [comment]
Please attach a screenshot.

**Comment by judah69** [comment]
have two iPhone SE2 and iPhone 14 Pro and I am using the same program, but there is a problem only with the 14 Pro.

**Comment by piquark6046** [comment]
Hmm.. I have only iPhone SE2 and don't have your issue on the device.
Can you reinstall the userscript manager app?

**Comment by judah69** [comment]
I already reinstalled the user script and AdGuard.

**Comment by piquark6046** [comment]
Do you have a Windows device?

**Comment by judah69** [comment]
I use AdGuard and Namu Link on Windows devices and S23, but they work well. The problem only appears on the 14 Pro.

**Comment by piquark6046** [comment]
Can you tell me what is its iOS version?

**Comment by judah69** [comment]
17.3

**Comment by judah69** [comment]
I tried rebooting the iPhone 14 Pro and using only Wi-Fi or only cellular, but it didn't work.

**Comment by judah69** [comment]
se2 is 17.1

14 pro is 17.3

**Comment by piquark6046** [comment]
I use iPhone SE2 with iOS 17.3.
By the way, do you use any VPN service for your iPhone SE2?

**Comment by judah69** [comment]
no i’am not use vpn

**Comment by FiestaLake** [comment]
Can confirm this also happens in the Desktop Firefox / ViolentMonkey setup.

**Comment by piquark6046** [comment]
@FiestaLake Do you use a VPN service?

**Comment by FiestaLake** [comment]
@piquark6046
No, I only use NextDNS as DNS-Over-TLS setup with KT as an ISP.
And found that it only happens on the senkawa theme, guessing you're gonna provide support for the new espejo theme only?

**Comment by piquark6046** [comment]
> And found that it only happens on the senkawa theme, guessing you're gonna provide support for the new espejo theme only?

Yes. Because umanle S.R.L. has stopped supporting senkawa theme, NamuLink userscript does not support Senkawa theme. Only espejo theme is supported.
 - https://gall.dcinside.com/mgallery/board/view/?id=namuwiki&no=80907&page=1

**Comment by piquark6046** [comment]
@FiestaLake 
> And found that it only happens on the senkawa theme, guessing you're gonna provide support for the new espejo theme only?

Yes. Because umanle S.R.L. has stopped supporting senkawa theme, NamuLink userscript does not support Senkawa theme. Only espejo theme is supported.
 - https://gall.dcinside.com/mgallery/board/view/?id=namuwiki&no=80907&page=1

**Comment by piquark6046** [comment]
@judah69 Are you same case with https://github.com/List-KR/NamuLink/issues/30#issuecomment-1916353053?

**Comment by Howu48** [comment]
same issue here.

iPhone / iOS 17.3
safari
no matter private relay is on or off
default espejo theme (never login)

**Comment by piquark6046** [comment]
@Howu48 Please attach a screenshot.

**Comment by Howu48** [comment]
> @Howu48 Please attach a screenshot.

![image](https://github.com/List-KR/NamuLink/assets/75659325/75c2ac64-fbae-4fb4-bd35-0072c2f1eced)

**Comment by piquark6046** [comment]
@Howu48 Where do you live (country)?

**Comment by Howu48** [comment]
> @Howu48 Where do you live (country)?

in Korea (south ofcourse)

**Comment by piquark6046** [comment]
@Howu48 @judah69 Some logic in NamuLink userscript are updated. Please try new version.

**Comment by FiestaLake** [comment]
Your recent changes break loading a document's history, my document box, etc...
Running Firefox with latest ViolentMonkey.

**Comment by piquark6046** [comment]
@FiestaLake 
Please tell me what is your environment (desktop or mobile).

**Comment by FiestaLake** [comment]
> @FiestaLake Please tell me what is your environment (desktop or mobile).

Desktop Firefox

**Comment by piquark6046** [comment]
What is your ASN?

**Comment by FiestaLake** [comment]
> What is your ASN? To see your ASN, please visit https://ipinfo.io/.

AS4766

**Comment by piquark6046** [comment]
It loads correctly on my side. Please attach a screenshot and console tab of your DevTool.

**Comment by FiestaLake** [comment]
https://youtu.be/0cItjOO1toM
![Screenshot from 2024-02-14 00-13-07](https://github.com/List-KR/NamuLink/assets/68979855/2b5513b0-cfad-4337-bcdd-88ebb42d6773)

**Comment by piquark6046** [comment]
It seems that you do not use a Firefox web browser. I think that it is [GNOME Web](https://apps.gnome.org/Epiphany/).

**Comment by FiestaLake** [comment]
> It seems that you do not use Firefox web browser. I think that it is [GNOME Web](https://apps.gnome.org/Epiphany/).

It's firefox-gnome-theme.

**Comment by piquark6046** [comment]
Can you record new screencast the behavior with console tab of Dev Tool again?

**Comment by piquark6046** [comment]
@FiestaLake Please test the following version:
https://github.com/List-KR/NamuLink/raw/update-20240213-1/NamuLink.user.js

**Comment by FiestaLake** [comment]
> @FiestaLake Please test the following version: https://github.com/List-KR/NamuLink/raw/update-20240213-1/NamuLink.user.js

I guess it's fixed on that version.

**Comment by FiestaLake** [comment]
> @FiestaLake Please test the following version: https://github.com/List-KR/NamuLink/raw/update-20240213-1/NamuLink.user.js

But my document box (내 문서함) is still broken.
